src/EmergencyEvaluator/Transcripter.class.st

```diff
@@ -223,7 +223,7 @@ Transcripter >> readEvalPrint [
 				ifTrue: [okToRevert
 						ifTrue: [(self confirm: 'Revert: ' , RecentMessageList uniqueInstance lastEntry asString , ' ?')
 								ifTrue: [RecentMessageList uniqueInstance revertLastMethodSubmission.
-									self cr; show: 'reverted: ' , RecentMessageList uniqueInstance lastEntry asString.
+									self cr; show: 'reverted: '; nextPutAll: RecentMessageList uniqueInstance lastEntry asString.
```

Comment:
Hmm, it should be the other way around: `nextPutAll:` first, then `show:`, or twice `nextPutAll:` followed by `flush`

Reply:
Ohh ... yes. Need to change it once there is a next time slot